app/views/layouts/application.html.erb

```diff
@@ -22,44 +54,57 @@
 
           <% if current_user %>
             <li><%= link_to 'Logout', logout_path, method: :delete, data: {confirm: 'Are you sure?'} %></li>
+            <li><%= link_to 'Add a Playspace', new_playspace_path %></li>
           <% else %>
             <li><%= link_to 'Sign up', new_user_path %></li>
             <li><%= link_to 'Login', login_path %></li>
           <% end %>
       </ul>
     </div>
-    <!-- /#sidebar-wrapper -->
+    <!-- /#sidebar-wrapper left-->
+
+    <!-- Sidebar right -->
+    <div id="sidebar-wrapper-right" class="hide">
+      <ul class="sidebar-nav">
+        <li class="sidebar-brand">PlaySpace</li>
+        <li><a href="#">Search</a></li>
+      </ul>
+    </div>
+    <!-- /#sidebar-wrapper right-->
+
     <!-- Page Content -->
-    <div id="page-content-wrapper">
-      <nav class="navbar navbar-default navbar-fixed-top">
-        <div class="container-fluid">
-          <div class="navbar-header">
-            <a class="navbar-brand">PlaySpace</a>
-            <button href="#menu-toggle" id="menu-toggle" type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false">
-              <span class="sr-only">Toggle navigation</span>
-              <span class="icon-bar"></span>
-              <span class="icon-bar"></span>
-              <span class="icon-bar"></span>
-            </button>
-          </div>
-        </div>
-      </nav>
+    <div id="test" class="page-content-wrapper">
       <div class="container-fluid">
         <div class="row">
           <%= yield %>
         </div>
       </div>
     </div>
-    <!-- /#page-content-wrapper -->
   </div>
-  <!-- /#wrapper -->
+  <!-- /#page-content-wrapper -->
 
   <!-- Menu Toggle Script -->
   <script>
+    if (window.location.pathname == "/about"){
```

Comment:
you shouldn't have code that runs on every page and conditionally executes depending if you are on a particular page:

It's much better to rename [staticPages.coffee](https://github.com/wdi-hk-9/Project-PlaySpace/blob/master/app%2Fassets%2Fjavascripts%2Fstatic_pages.coffee) to `staticPages.js` and add:

```js
var About = function() {
  $("#sidebar-wrapper-right").removeClass("hide");
  $("#menu-toggle2").removeClass("hide");
  $("#menu-toggle3").removeClass("hide");
}
```

Then from https://github.com/wdi-hk-9/Project-PlaySpace/blob/master/app%2Fviews%2Fstatic_pages%2Fabout.html.erb you can add:

```html
<script>
  About()
</script>
```

And it will execute only that code